packages/cli/src/main.ts

```diff
@@ -64,13 +64,6 @@ export const main: () => void = async () => {
       (argv) => handleDataDir(argv.dataDir),
       (argv) => setOptions(argv),
     ]).argv;
-
-  if (promise instanceof Promise) {
-    promise.catch((error) => {
```

Comment:
This didn't seem to actually do anything as far as I could tell

Reply:
I think I had typing issues with this before. I think it's good to remove it if it works without.

Reply:
I tested a couple of cases (error thrown in handler, error parsing args or command), and as far as I can tell it has no effect.